TMGToolbox/src/network_editing/network_comparison/copy_transit_lines.py

```diff
@@ -503,8 +521,16 @@ def _Execute(self):
             vehicleTable = self._LoadVehicleCorrespondenceFile(sourceNetwork, targetNetwork)
             print "Loaded vehicle correspondence table"
             
+            if self.LinksAllowedForModification is not None:
+                if self.TargetScenario.extra_attribute(self.linkModAttrID) is not None:
+                    self.TargetScenario.delete_extra_attribute(self.linkModAttrID)
+                self.TargetScenario.create_extra_attribute('LINK', self.linkModAttrID)
+                networkCalculationTool(self._getLinkCalcSpec(), self.TargetScenario)
+                targetNetwork = self.TargetScenario.get_network()
```

Comment:
Could we reorder the logic here so that the network will only be loaded once?  I don't believe that any of the above logic needs to happen before this.